.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,47 @@
+name: 🐞 Bug Report
+description: Tell us about something that's not working the way we (probably) intend.
+labels: ["Kind:Bug", "State:Triage"]
+body:
+  - type: input
+    id: elixir-version
+    attributes:
+      label: Elixir version
+      description: Use `elixir -v` to find the Elixir version.
+    validations:
+      required: true
+
+  - type: input
+    id: erlang-version
+    attributes:
+      label: Erlang version
+      description: Use `elixir -v` to find the Erlang version.
+    validations:
+      required: true
```

Can we merge these into a single field, in the interest of streamlining the process as much as possible?

```suggestion
  - type: input
    id: elixir-and-otp-version
    attributes:
      label: Elixir and Erlang/OTP versions
      description: Paste the output of `elixir -v` here.
    validations:
      required: true
```